tools/licenses/lib/main.dart

```diff
@@ -2150,6 +2150,10 @@ class RepositoryGarnetDirectory extends RepositoryDirectory {
   @override
   bool shouldRecurse(fs.IoNode entry) {
     return entry.name != 'bin'
+        && entry.name != 'docs'
+        && entry.name != 'examples'
+        && entry.name != 'go'
+        && entry.name != 'lib'
```

Comment:
lib isn't shipped?

Reply:
The libraries that we link to are in garnet/public/lib

Reply:
`//garnet/lib` is meant to be private.  We shouldn't have dependencies on anything outside the `//garnet/public` directory (and, transitively, `public` shouldn't have any such dependencies either).

Reply:
Private vs public isn't the issue here, it's whether any of the files in `lib/` in any way influence the contents of the final binary.

Reply:
`public` doesn't have any dependencies in `//garnet` on anything outside of `public`.  In particular, `public` doesn't depend on `lib` or `bin`.